### DISCUSSION TITLE
Is there a way of checking if the component was lazy loaded?

### DISCUSSION THREAD

**Discussion by danjdewhurst** [livewire/livewire#7658] [1. Help]
I'm pushing to a blade stack from a Livewire component which works fine. but when the component is lazy loaded it does not update the blade stack, which is understandable.

I want to find a way to detect if the component was lazy-loaded and then deal with the stack another way. I can't find a method or property to check this though.

Any ideas?

Thanks

Update: `@teleport` does what I need, but it'd still be useful to get an answer.

**Comment by calebporzio** [comment]
Yeah there's currently no surface level API.

Glad teleport worked for you.

I think between @script @assets and @teleport most needs are covered.

What we're you picturing, like "$this->isLazy"?

**Reply by danjdewhurst** [reply]
> Yeah there's currently no surface level API.
> 
> Glad teleport worked for you.
> 
> I think between @script @assets and @teleport most needs are covered.
> 
> What we're you picturing, like "$this->isLazy"?

Yep, exactly that. As you said not a priority but would be nice in the future. Thanks.
